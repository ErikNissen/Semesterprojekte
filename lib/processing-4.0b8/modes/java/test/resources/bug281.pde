version https://git-lfs.github.com/spec/v1
oid sha256:b2ebcf69c2835e4b95cb6a56c92eda54d536ef0934b7c8f3d96218da32174ed4
size 64
