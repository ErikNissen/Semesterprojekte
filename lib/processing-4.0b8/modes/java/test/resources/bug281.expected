version https://git-lfs.github.com/spec/v1
oid sha256:deec0f1b5a21798be916799b13459249abc1e013d9b100aeb4379aa3e555e4e0
size 755
